index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Nacho Sala</title>
+	<link rel="stylesheet" href="style.css">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" 
+rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" 
+rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+	<header id="appbar">
+    <div id="toolbar"> 
+      <div id="header-logo">Nacho Sala</div>
+      <img src="./images/menu.png" alt="Menu">
+    </div>
+	</header>
+	<section id="headline">
+		<h1 id="main-header">Hey There. <br> I'm Nacho</h1>
+		<h2 id="sub-main-header">I am a Software Developer</h2>
+		<p id="main-paragraph">I can help you build a product, feature or website. Look through some of my 
+			work and experience! If you like what you see and have a project you need coded, 
+			don’t hesitate to contact me.
+    </p>
+    <section id="social-links">
+      <img src="./images/social-links/github.png" alt="Github">
+      <img src="./images/social-links/linkedin.png" alt="LinkedIn">
+      <img src="./images/social-links/angelist.png" alt="AngelList">
+      <img src="./images/social-links/twitter.png" alt="Twitter">
+      <img src="./images/social-links/medium.png" alt="Medium">
+    </section>
```

Comment:
- [ ] The social media icons should be grouped in a list. Kindly use the `<ul>` tag instead of the section one.
- [ ] Each media icon is a clickable link. Kindly use the correct <a> tag.